tests/api-testing/tests/test_cipher.py

```
@@ -0,0 +1,984 @@
+
+    # Search decrypted the cipher key
+    json_search = {
+    "query": {
+        "sql": sql_query,
+        "start_time": one_min_ago,
+        "end_time": end_time,
+        "sql_mode": "full",
+        "from":0,
+        "size":100,
+        "quick_mode":False,
+        "sql_mode":"full"
```

Ruff (0.8.2)

Dictionary key literal `"sql_mode"` repeated

Remove repeated key literal `"sql_mode"`

(F601)